lib/css/components/_stacks-banners.less

```diff
@@ -73,9 +73,8 @@
 
 //  --  When we want to keep hero content capped
 .s-banner--container {
-    #stacks-internals #load-config();
     position: relative;
     width: 100%;
-    max-width: @body-site-width;
+    max-width: 1060px; // TODO consider creating a CSS variable for this
```

Comment:
~~Definitely move this out to a variable. If we don't intend on allowing consumers to change this value, let's at least make it a Less variable so we can do math on it.~~

Wait, we're literally only using this variable in this one spot? Let's see if we can reuse a different variable instead. Perhaps we could use `@s-full`? That's what `.wmx12` and the topbar container uses by default.

Reply:
I'm not sure why this was set to `1060px` to begin with. That value doesn't map to any existing value, so the banner will never be aligned with other content. I kept that value in place just to keep from changing **any_** values in this PR, but I think we'd be safe to tweak it.

How about `@s-step * 10`? That would be equivalent to `.wmx10` which is roughly `1054px`.

Reply:
I think mimicking `.wmx10` would be an excellent compromise. We can always revisit later if we want to.